server/seeder/modules/fileComposer/fileComposer.spec.js

```diff
@@ -1,15 +1,35 @@
 import FileComposer from "./FileComposer";
-import FileComposerConsumer from "./fileComposerConsumer";
-jest.mock("./fileComposer");
 
 beforeEach(() => {
-  // Clear all instances and calls to constructor and all methods
-  FileComposer.mockClear();
 });
 
 describe("FileComposer", () => {
-  it("consumer should call FileComposer class constructor at least once", () => {
-    const fileComposerConsumer = new FileComposerConsumer();
-    expect(FileComposer).toHaveBeenCalledTimes(1);
+  it("should construct correctly deep tree", () => {
```

Comment:
Cool 👍 
Thanks for these examples, I see where it could lead when writing these seeder tests